### DISCUSSION TITLE
I did a Tutorial for a Apache Setup on ArchLinux

### DISCUSSION THREAD

**Discussion by muellerjoel** [jesec/flood#464] [Show and tell]
Type: Documentation Issue

## Summary

https://joelmueller.ch/latest-flood-webui-with-rtorrent-and-apache-on-archlinux/

You can use it for your documentary if you want or post the link in your wiki

Cheers

**Comment by jesec** [comment]
Great work. We have a section in the discussions that show the awesome works.
